modules/nodes-cluster-enabling-features-about.adoc

```diff
@@ -21,8 +21,18 @@ a|Enables Technology Preview features that are not part of the default features.
 
 The following Technology Preview features are enabled by this feature set:
 
-* link:https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/#certificate-rotation[RotateKubeletServerCertificate]. Enables the rotation of the server TLS certificate on the kubelet.
-* link:https://kubernetes.io/docs/concepts/policy/pid-limiting/#pod-pid-limits[Pod PID limits (SupportPodPidsLimit)]. Enables limiting process IDs (PIDs) in pods.
+* Azure Disk CSI Driver Operator. Enables provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Microsoft Azure Disk Storage.
+* VMware vSphere CSI Driver Operator. Enables provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) VMware vSphere driver for Virtual Machine Disk (VMDK) volumes.
+* CSI automatic migration. Enables automatic migration for supported in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers. Supported for:
+** Amazon Web Services (AWS) Elastic Block Storage (EBS)
+** OpenStack Cinder
+** Azure Disk
+** Google Compute Engine Persistent Disk (in-tree)
+** Google Cloud Platform Persistent Disk (CSI).
+* External Cloud Provider. Enables using an external cloud-controller-manager rather than the in-tree cloud controller. Available as a Technology Preview for: 
+** Amazon Web Services (AWS)
+** Microsoft Azure
+** Red Hat OpenStack Platform (RHOSP)
 * `InsightsOperatorPullingSCA`. Enables importing of RHEL Simple Content Access (SCA) certificates from {console-redhat-com}.
```

Comment:
Is this really for the 4.9 docs please? If so then this is correct, but if this is for the 4.10 then this lines needs to be removed please.

Reply:
OK thanks. No problem. Then it's correct.